snforge_std/src/cheatcodes.cairo

```diff
@@ -35,13 +42,24 @@ fn stop_roll(target: CheatTarget) {
     cheatcode::<'stop_roll'>(inputs.span());
 }
 
-fn start_prank(target: CheatTarget, caller_address: ContractAddress) {
+fn prank(target: CheatTarget, caller_address: ContractAddress, span: CheatSpan) {
+    assert!(span != CheatSpan::Number(0), "CheatSpan must be greater than 0");
+
     let mut inputs = array![];
     target.serialize(ref inputs);
+    span.serialize(ref inputs);
     inputs.append(caller_address.into());
     cheatcode::<'start_prank'>(inputs.span());
 }
 
+fn prank_once(target: CheatTarget, caller_address: ContractAddress) {
+    prank(target, caller_address, CheatSpan::Number(1))
+}
+
+fn start_prank(target: CheatTarget, caller_address: ContractAddress) {
+    prank(target, caller_address, CheatSpan::Indefinite)
+}
+
 fn stop_prank(target: CheatTarget) {
     let mut inputs = array![];
     target.serialize(ref inputs);
```

Comment:
I would just remove the `start_prank` (and `prank_once`) because the interface is too cluttered now

Reply:
The `stark_prank` should stay for sure, we don't want to break the interface when it is not needed

Reply:
We are still in `0. ...` so breaking the interface is not as bad as it could.
Maybe we could deprecate `start_prank`? and remove it in the next versions?